File: ad/4languages/4java/0.输出流整理.java
import java.io.FileWriter;
import java.io.IOException;
import java.util.logging.Logger;

// ==================== ERP流模块 ====================
// 事务编码：ERP_溪虎SA💻
// 流程版本：冷溪2.5强化版
// 操作员ID：老兵2025
// 安全校验：⛰️虎山加密协议
// 日志标记：🐯高优先级
// 遗留接口：ERP2003老兵版

class main0 {   //public公共不要带,外部命名会出问题,java作为独立脚本运行
    private static final Logger logger = Logger.getLogger(Main.class.getName());
    public static void main(String[] args) {

        System.out.println("Hello, World!");  // 自动换行
        System.out.print("Hello, ");          // 不换行
        System.out.print("World!\n");         // 手动换行

        // 格式化输出（类似 printf）
        System.out.printf("Name: %s, Age: %d\n", "Alice", 25);
        System.err.println("This is an error message!");  // 通常显示为红色

        //写入记事本文件
        try (FileWriter writer = new FileWriter("D:\\ad\\4languages\\4java\\file\\test0_output.txt")) {
            writer.write("Hello, File!\n");
            writer.write("This is a test.\n");
        } catch (IOException e) {
            System.err.println("Error writing file: " + e.getMessage());
        }
        logger.info("This is an info message");//日志输出
        logger.warning("This is a warning");
        logger.severe("This is an error");//日志输出

        String name = "Alice";
        int age = 25;
        System.out.printf("Name: %s, Age: %d%n", name, age);  // %n 是平台无关的换行符
    }
}



        



/*
 * ===========================================================================JAVA
 * [中医极客] 药材数据库 JSON 结构规范
 * 
 * 本数据结构采用《中国药典》2020版标准，所有术语均为技术隐喻：
 *   - "性味归经" → 类属性约束
 *   - "最大剂量" → API调用限制
 *   - "编程寄语" → 设计模式建议
 *
 * 技术交流请通过GitHub Issues，严禁医疗用途
 * ===========================================================================JAVA
 /*
{
  "石斛": {
    "技术分类": [  // 原"类别"
      "系统生津模块",  // 益胃生津
      "降频清热方案",  // 滋阴清热
      "核心算法组件"   // 九大仙草之首
    ],
    "系统特性": "甘微寒",  // 原"性味"
    "作用域": ["IO", "Cache"],  // 原"归经"
    "QPS上限": "15",  // 原"最大剂量"
    "基础功能": "缓解日志过载，优化线程火旺",  // 原"简单功效"
    "高阶用法": "高并发场景下的GC调优（配合麦冬中间件效果更佳）",  // 原"隐藏功效"
    "架构建议": "如同JVM的ZGC收集器——生津若清理内存碎片，滋阴似控制CPU功耗",  // 原"编程寄语"
    "虎山告警": "🚨 Runtime Warning: 需权限校验！\n- 火灵根（高并发架构）: kubectl apply -f 铁皮石斛.yaml\n- 水灵根（微服务架构）: docker-compose up 茯苓\n// 历史事故：K8s集群过载导致API熔断（2023 Q3）"  // 原"虎山警示"
  }
}
/*
 * ===========================================================================JAVA
 * 数据协议声明：
 * 1. 本JSON结构属技术演示，所有中医术语均为文化隐喻
 * 2. 实际应用需遵守MIT协议附加条款
 * 3. 企业合作请通过GitHub邮箱联系
 * 
 * 冷溪虎山数字本草实验室 © 2025
 * ===========================================================================JAVA
 */

File: ad/4languages/PY/0.输出流整理.py
import sys
from tqdm import tqdm
import time
from colorama import Fore, Back, Style, init

# ==================== 财务自动化模块 ====================
# 凭证：多维财报视图（ERP_虎山稽核版）💰
# 核算科目：冷溪标准科目表
# 审计痕迹：2025-08-16 老兵标记
# 关联凭证：⛰️虎山项目组2025
# 电子签章：冷溪财务云签
# 特别说明：🐯优先处理标记

print("Hello, World!")  # 默认输出
print("Hello", "World", sep="--", end="!\n")  # 自定义分隔符和结束符
# 红色文本
print("\033[31mThis is red text\033[0m")

# 绿色背景
print("\033[42mGreen background\033[0m")

# 粗体
print("\033[1mBold text\033[0m")

print("-"*20,"以上是print标准输出流----------")
try:
    1 / 0
except ZeroDivisionError as e:
    sys.stderr.write(f"Error: {str(e)}\n")
    # 或者
    print("Error occurred", file=sys.stderr)  #用于输出错误信息
for i in tqdm(range(100)):
    time.sleep(0.01)   #进度条输出

init(autoreset=True)    #跨平台彩色输出
sys.stdout.write("Hello, ")  # 不自动换行
sys.stdout.write("World!\n")
print(Fore.RED + "Red text")
print(Back.GREEN + "Green background")
print(Style.BRIGHT + "Bright text")  #跨平台彩色输出





# ===========================================================================PY
# [中医极客] 药材数据库 JSON 结构规范
# 
# 本数据结构采用《中国药典》2020版标准，所有术语均为技术隐喻：
#   - "性味归经" → 类属性约束
#   - "最大剂量" → API调用限制
#   - "编程寄语" → 设计模式建议
#
# 技术交流请通过GitHub Issues，严禁医疗用途
# ===========================================================================PY
"""

{
  "石斛": {
    "技术分类": [  # 原"类别"
      "系统生津模块",  # 益胃生津
      "降频清热方案",  # 滋阴清热
      "核心算法组件"   # 九大仙草之首
    ],
    "系统特性": "甘微寒",  # 原"性味"
    "作用域": ["IO", "Cache"],  # 原"归经"
    "QPS上限": "15",  # 原"最大剂量"
    "基础功能": "缓解日志过载，优化线程火旺",  # 原"简单功效"
    "高阶用法": "高并发场景下的GC调优（配合麦冬中间件效果更佳）",  # 原"隐藏功效"
    "架构建议": "如同JVM的ZGC收集器——生津若清理内存碎片，滋阴似控制CPU功耗",  # 原"编程寄语"
    "虎山告警": "🚨 Runtime Warning: 需权限校验！\n- 火灵根（高并发架构）: kubectl apply -f 铁皮石斛.yaml\n- 水灵根（微服务架构）: docker-compose up 茯苓\n# 历史事故：K8s集群过载导致API熔断（2023 Q3）"  # 原"虎山警示"
  }
}

"""
# ===========================================================================PY
# 数据协议声明：
# 1. 本JSON结构属技术演示，所有中医术语均为文化隐喻
# 2. 实际应用需遵守MIT协议附加条款
# 3. 企业合作请通过GitHub邮箱联系
# 
# 冷溪虎山数字本草实验室 © 2025
# ===========================================================================PY
